File: U3P03-PHP-Catalogo/Cancion.php
<?php
class Cancion
{
    private $ID_CANCION;
    
    private $ID_ALBUM;
    
    private $NOMBRE;
    
    private $ID_AUTOR;
    
    private $DURACION;
   
    
    
    public function __toString()
    {
        // $imgs="/U3P03-PHP-Catalogo/img/";    . " <img src=" . $imgs . $this->imagen . " height=150 width=150>"
        
        $k=$this->ID_CANCION . " " . $this->ID_ALBUM . " " . $this->NOMBRE . " " . $this->ID_AUTOR . " " . $this->DURACION ;
        return $k;
    }
    
    public function getIDCANCION() {
        return $this->ID_CANCION;
        
    }
    
    public function getIDALBUM() {
        return $this->ID_ALBUM;
        
    }
    
    public function getNombre() {
        return $this->NOMBRE;
        
    }
    
    public function getIDAUTOR() {
        return $this->ID_AUTOR;
        
    }
    
    public function getDURACION() {
        return $this->DURACION;
        
    }
    
  /*  public function getImagen() {
        $imgs="/U3P03-PHP-Catalogo/img/";
        
        return "<img src=" . $imgs . $this->imagen . " height=150 width=150>";
      } */      
}
?>

File: U5P03-Java-Autenticacion/src/model/Cancion.java
package model;

public class Cancion {

	private int ID_CANCION, ID_ALBUM, ID_AUTOR;
	private String NOMBRE;
	private float DURACION;
	public int getID_CANCION() {
		return ID_CANCION;
	}
	public void setID_CANCION(int iD_CANCION) {
		ID_CANCION = iD_CANCION;
	}
	public int getID_ALBUM() {
		return ID_ALBUM;
	}
	public void setID_ALBUM(int iD_ALBUM) {
		ID_ALBUM = iD_ALBUM;
	}
	public int getID_AUTOR() {
		return ID_AUTOR;
	}
	public void setID_AUTOR(int iD_AUTOR) {
		ID_AUTOR = iD_AUTOR;
	}
	public String getNOMBRE() {
		return NOMBRE;
	}
	public void setNOMBRE(String nOMBRE) {
		NOMBRE = nOMBRE;
	}
	public float getDURACION() {
		return DURACION;
	}
	public void setDURACION(float dURACION) {
		DURACION = dURACION;
	}
	public Cancion(int iD_CANCION, int iD_ALBUM, int iD_AUTOR, String nOMBRE, float dURACION) {
		super();
		ID_CANCION = iD_CANCION;
		ID_ALBUM = iD_ALBUM;
		ID_AUTOR = iD_AUTOR;
		NOMBRE = nOMBRE;
		DURACION = dURACION;
	}

}
